README.md

```diff
@@ -100,125 +60,57 @@ After you're finished please remove all the comments and instructions!
 
 ### Key Features <a name="key-features"></a>
 
-> Describe between 1-3 key features of the application.
-
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
+- **[query to list specific animal name]**
+- **[query to list specific animal per date]**
+- **[query to create the animal table]**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> Add a link to your deployed project.
-
-- [Live Demo Link](<replace-with-your-deployment-URL>)
+- [will be available soon](<replace-with-your-deployment-URL>)
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- Hi :clap:, Since you've not deployed the project, kindly remove the `live demo` section from the top navigation and from the readme body to make the readme more professional.